### ISSUE TITLE
Article list should be reverse chronological

### ISSUE THREAD

**Issue by eccogrinder** [gnosis/tabula#263]
**Is your feature request related to a problem? Please describe.**
I think articles should be displayed reverse chronologically to continue improving navigation and UX.

**Describe the solution you'd like**
Display articles in reverse chronological order.

**Describe alternatives you've considered**
Option to highlight or pin important articles.

**Comment by auryn-macmillan** [comment]
Are they not displayed in reverse chronological order now? I had kind of taken this for granted.

**Comment by eccogrinder** [comment]
Yeah, and actually, they are currently chronological instead of reverse chronological.
